jdaviz/configs/mosviz/tests/test_data_loading.py

```diff
@@ -222,8 +222,11 @@ def test_load_single_image_multi_spec(mosviz_helper, mos_image, spectrum1d, mos_
 
     label_mouseover._viewer_mouse_event(spec2d_viewer,
                                         {'event': 'mousemove', 'domain': {'x': 10, 'y': 100}})
+
+    # Note: spectra2d Wave loaded in meters, but we respect one spectral unit, so the meters in
+    # converted to Angstrom (the spectra1d spectral unit).
```

Comment:
Can you explain this a little more?  Is it that a previous data entry was in angstroms or just that that is the default spectral display unit?

```suggestion
    # Note: spectra2d Wave loaded in meters, but we respect one spectral unit, so the meters is
    # converted to Angstrom (the spectra1d spectral unit).
```

Reply:
So the loaded 1D and 2D spectra have different native spectral units, with one in meters and one in angstroms and app-wide we're respecting one choice for spectral unit.